modules/core/shared/src/main/scala/util/Pool.scala

```diff
@@ -49,11 +49,11 @@ object Pool {
    * @param recycler a cleanup/health-check to be done before elements are returned to the pool;
    *   yielding false here means the element should be freed and removed from the pool.
    */
-  def of[F[_]: Concurrent: Trace, A](
-    rsrc:  Resource[F, A],
+  def of[F[_]: Concurrent, A](
+    rsrc:  Trace[F] => Resource[F, A],
```

Comment:
This is binary incompatible, right?

Reply:
This should be fixed now with a name change for this, and introducing a universal scoped version thats equivalent to the previous of.